sympy/series/tests/test_residues.py

```diff
@@ -80,7 +80,7 @@ def test_issue_14037():
 def test_issue_21176():
     f = x**2*cot(pi*x)/(x**4 + 1)
     assert residue(f, x, -sqrt(2)/2 - sqrt(2)*I/2).together() == \
```

Comment:
It looks like `cancel().together(deep=True)` would give a better looking result.